### PR TITLE
ucsc/wigtobigwig: add meta map

### PR DESCRIPTION
**Pull request by projectoriented**
## PR checklist

This PR adds meta map to ucsc/wigtobigwig module to standardize with all other modules that hold meta values.

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

**Comment by projectoriented** [comment]
reminder to self: when this is merged, reverse tiddit/cov in qc_bam.nf

File: tests/modules/ucsc/wigtobigwig/test.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 - name: ucsc wigtobigwig test_ucsc_wigtobigwig
-  command: nextflow run ./tests/modules/ucsc/wigtobigwig -entry test_ucsc_wigtobigwig -c ./tests/config/nextflow.config -c ./tests/modules/ucsc/wigtobigwig/nextflow.config
+  command: nextflow run tests/modules/ucsc/wigtobigwig -entry test_ucsc_wigtobigwig -c tests/config/nextflow.config
```
Comment (drpatelh) [inline tests/modules/ucsc/wigtobigwig/test.yml:2]:
```suggestion
  command: nextflow run tests/modules/ucsc/wigtobigwig -entry test_ucsc_wigtobigwig -c tests/config/nextflow.config -c ./tests/modules/ucsc/wigtobigwig/nextflow.config
```

File: modules/ucsc/wigtobigwig/main.nf [hunk 2]
```diff
@@ -10,21 +10,22 @@ process UCSC_WIGTOBIGWIG {
...
     input:
-    path wig
+    tuple val(meta), path(wig)
     path sizes
 
     output:
-    path "*.bw"        , emit: bw
-    path "versions.yml", emit: versions
+    tuple val(meta), path("*.bw")  , emit: bw
+    path "versions.yml"            , emit: versions
```
Comment (drpatelh) [inline modules/ucsc/wigtobigwig/main.nf:18]:
```suggestion
    tuple val(meta), path("*.bw"), emit: bw
    path "versions.yml"          , emit: versions
```

**Review by drpatelh — APPROVED** [review]
👌🏽

**Pull request merged**